config/locales/en.yml

```diff
@@ -1,3 +1,10 @@
 en:
+  games:
```

Comment:
Think this guy (and the one below) isn't used anymore with the more blanket 404 handling.